.github/workflows/test.yml

```diff
@@ -22,10 +22,17 @@ jobs:
         run: ./scripts/install_foundry.sh
 
       - name: Run unit tests
+        id: sol_unit_tests
         run: |
           export PATH=$PATH:$HOME/.foundry/bin
           cd contracts/
           forge test -vvv
+      
+      - name: Check Solidity Formatting
+        if: success() || failure()
+        run: |
+          export PATH=$PATH:$HOME/.foundry/bin
+          ./scripts/lint.sh --sol-format-check
```

Comment:
This seems out of place in the test job. Can we move to the linter job, or better yet a standalone format check job?